CONTRIBUTING-detailed-instruction.md

```diff
@@ -0,0 +1,100 @@
+	```Bash
+	$ git remote add upstream https://github.com/scalabridge/curriculum.git
+
+	Press `Enter` and git will create your local clone
+    ```
+6. Verify your remote repositories by entering `git remote -v`. You should see the URL for your fork as `origin`, and the URL for the scalabridge repository as `upstream`:
+```bash
+$ git remote -v
+origin  https://github.com/YOUR-USERNAME/curriculum.git (fetch)
+origin  https://github.com/YOUR-USERNAME/curriculum.git (push)
+upstream        https://github.com/scalabridge/curriculum.git (fetch)
+upstream        https://github.com/scalabridge/curriculum.git (push)
+```
+
+
+
+### Pull Request Work Flow
+1. **[Syncing your fork](https://help.github.com/articles/syncing-a-fork/):** Start with your local and forked repository being up-to-date with scalabridge/master:
+	1. Open terminal or Git Bash and change working directory to your local repository
+	2. Fetch the branches and their respective commits from the upstream repository. Commits to `master` will be stored in a local branch, `upstream/master`. 
+	3. Then merge the changes from `upstream/master` into your local `master` branch. 
+	4. Finally, update your forked repository on GitHub by pushing the changes to `origin`:
+	```bash
+	$ git fetch upstream  # Fetch the branches and their respective commits
+	From https://github.com/scalabridge/curriculum
+	 * [new branch]      master     -> upstream/master
+	# merge upstream/master into local master
+	$ git checkout master
+	$ git merge upstream/master
+	# push changes to your forked repository
+	$ git push origin
+	```
+2. **Create a topic branch** on your local repository.
+	1. From the repository directory
+	2. `git checkout master` to make master the base
+	3. `git checkout -b TOPIC-NAME`, where TOPIC-NAME is an appropriate name for branch, e.g "contributing" 
+3. **Make your changes**: hack, hack, edit, hack, edit, ...
+4. **Add and commit** changes to your topic branch. Refer to the [Git Documentation](https://git-scm.com/docs) for details on [`add`](https://git-scm.com/docs/git-add) and [`commit`](https://git-scm.com/docs/git-add) usage:
+	1. Use the `git add` command to add content to the index that defines the snapshot that will be committed,
+	2. Use the `git commit` command to commit the index to the current working branch, or
+	3. Use `git commit -a` to add and commit all changes.
+	```bash
+	$ git add *                # adds all changes to index
+    $ git add CONTRIBUTING.md  # add single file to index
+	$ git commit -a            # adds all changes and commits
+	$ git commit -m "<commit message line here>"
+	```
+5. **Push your changes** to your forked repository. The first time you push your branch use the `git push -u` command as shown below. Subsequent pushes after further edits can use `git push origin` (assumes you are working in branch on local repository).
```

Comment:
I think you only need to do `git push` for subsequent pushes, since Git will know that the default remote is `origin`.